apps/src/sites/studio/pages/levels/show.js

```diff
@@ -55,6 +56,18 @@ function initPage() {
 
   const rubricFabMountPoint = document.getElementById('rubric-fab-mount-point');
   if (rubricFabMountPoint && experiments.isEnabled('ai-rubrics')) {
-    ReactDOM.render(<RubricFloatingActionButton />, rubricFabMountPoint);
+    const rubricData = getScriptData('rubric');
+    const reportingData = {
+      unitName: config.script_name,
+      courseName: config.course_name,
+      levelName: config.level_name,
```

Comment:
update reportingDataShape to include courseName?

Reply:
yes thank you!